src/Posts.tsx

```diff
@@ -65,8 +229,64 @@ class Posts extends React.Component<Props, State> {
           <div key={post.id} className="posts">
             <h5 className="post-topic">{post.topic}</h5>
             <h6 className="post-post">{post.post}</h6>
+            <Moment format="HH:mm DD.MM.YYYY">{post.created.toString()}</Moment>
```

Comment:
Please use smaller and not so legacy library when formatting dates :) This is good one https://date-fns.org/